docs/blog/2019-04-29-component-shadowing/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: What is Component Shadowing?
+date: 2019-04-29
+author: John Otander
+excerpt: "Gatsby Themes introduce a concept called Component Shadowing. This feature allows
+users to override a component in order to customize its rendering."
+tags:
+  - themes
+---
+
+Gatsby Themes introduce a concept called Component Shadowing. This feature allows
+users to override a component in order to customize its rendering. Component Shadowing
+is a powerful way to make one-off changes to a theme without having to fully eject.
+
+It's pretty common to end up with a theme where you're happy with 99% of the details
```

Comment:
the causality is a bit confusing here — how did you end up with this theme? I assume it's a theme you didn't author but installed? Could be clarified.